Mobster/app/src/main/java/Objects/Question.java

```diff
@@ -124,6 +124,10 @@ public int getIsFlagged() {
         return this.isFlagged;
     }
 
+    public void incrementFlagged() {
+        this.isFlagged++;
```

Comment:
Incrementing the flag to allow users to unflag if they flag by mistake also so that the admin can see the questions that have been flagged by multiple users and not just by 1 user